scripts/complete_liquidity_provision.js

```diff
@@ -105,15 +116,28 @@ module.exports = async (callback) => {
       callback("Error: Choose a smaller fleetSize, otherwise your payload will be to big for Infura nodes")
     }
 
-    console.log(`2. Deploying ${argv.fleetSize} trading brackets`)
-    const bracketAddresses = await deployFleetOfSafes(masterSafe.address, argv.fleetSize, true)
-    console.log("List of bracket traders in one line:", bracketAddresses.join())
-
-    // Sleeping for 3 seconds to make sure Infura nodes have processed all newly deployed contracts so that
-    // they can be awaited.
-    await sleep(3000)
+    let bracketAddresses
+    if (argv.brackets.length > 0) {
+      console.log("==> Skipping safe deployment and using brackets safeOwners")
+      bracketAddresses = argv.brackets
+      // Ensure that safes are all owned solely by masterSafe
+      await Promise.all(
+        bracketAddresses.map(async (safeAddr) => {
+          if (!(await isOnlySafeOwner(masterSafe.address, safeAddr))) {
```

Comment:
nit-pick: assert(await isOnlySafeOwner would be shorter.

Reply:
Yes, I like the shorted, but don't we want to be using callbacks here?